src/lib/button/button.scss

```diff
@@ -89,9 +94,9 @@
   z-index: 1;
 }
 
-// Applies a clearer border for high-contrast mode (a11y)
-@media screen and (-ms-high-contrast: active) {
-  .md-raised-button, .md-fab, .md-mini-fab {
-    border: 1px solid #fff;
+// Add an outline to make it more visible in high contrast mode.
+@include md-high-contrast {
+  [md-button], [md-raised-button], [md-icon-button], [md-fab], [md-mini-fab] {
+    border: solid 1px;
```

Comment:
Good point, I hadn't though about it. I'll give it a try and see how it works out. One nice side effect of using the border is that we don't _have_ to specify the color, it inherits from the text color.